tests/unittest/_torch/executor/test_mamba_cache_manager.py

```
@@ -543,6 +658,26 @@ def test_hybrid_models_default_to_v2_and_python_transceiver(monkeypatch):
         assert llm_args.cache_transceiver_config.transceiver_runtime == "PYTHON"
 
 
+def test_kimi_defaults_to_v2(monkeypatch: pytest.MonkeyPatch) -> None:
+    from tensorrt_llm._torch.models.modeling_kimi_linear import KimiLinearForCausalLM
+
+    monkeypatch.delenv("TRTLLM_USE_PY_MAMBA", raising=False)
+    monkeypatch.delenv("TLLM_MAMBA_MANAGER_PREFERENCE", raising=False)
+
+    llm_args = TorchLlmArgs(model="/tmp/dummy_model")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/dummy_model"

(S108)